src/Endpoints/Block.php

```diff
@@ -6,6 +6,7 @@
 use FiveamCode\LaravelNotionApi\Exceptions\NotionException;
 use FiveamCode\LaravelNotionApi\Exceptions\HandlingException;
 use FiveamCode\LaravelNotionApi\Entities\Collections\BlockCollection;
+use FiveamCode\LaravelNotionApi\Entities\Blocks\Block as BaseBlockEntity;
```

Comment:
Would "BlockEntity" as alias be clear enough?

Reply:
✅ done